examples/psa_crypto/example_hash.c

```diff
@@ -22,8 +22,9 @@
 
 #include "psa/crypto.h"
 
-static const uint8_t msg[] = "Hello World!";
-static const size_t msg_len = sizeof(msg)-1; // exclude NULL-byte
+/* certain PSA backends require the data to be in RAM rather than ROM */
```

Comment:
Very nit-picky, but without the context of the change, I think some ... context is missing ;-)

```suggestion
/* certain PSA backends require the data to be in RAM rather than ROM
 * so these values cannot be `const` */
```

Reply:
updated accordingly :)